packages/edit-site/src/components/global-styles/root-menu.js

```diff
@@ -33,10 +34,20 @@ function RootMenu() {
 	const hasShadowPanel = true; // useHasShadowPanel( settings );
 	const hasDimensionsPanel = useHasDimensionsPanel( settings );
 	const hasLayoutPanel = hasDimensionsPanel;
+	const hasBackgroundPanel = true; // useHasBackgroundPanel( settings );
```

Comment:
Can you elaborate on why we are hardcoding this?

Reply:
I am not sure exactly why, but the value of `backgroundImage` and `backgroundSize` is false here 🤔 .

Reply:
That can be investigated separately, I guess? Let's make sure we revert this change before merging

Reply:
Related: https://github.com/WordPress/gutenberg/pull/63216#discussion_r1740569717

Basically, I wonder if `useHasBackgroundPanel` should be updated and/or the logic of this panel refactored to include:

- the fact that there are controls for both the background image and the background color, which may depend on separate settings
- the fact that this panel can be used both for global site settings, and for block-level settings